pkg/controller/schedule_controller_test.go

```diff
@@ -259,41 +262,41 @@ func parseTime(timeString string) time.Time {
 func TestGetNextRunTime(t *testing.T) {
```

Comment:
It looks like all the remaining tests in this file use the struct literals.

Reply:
Got those updated